hbase-shell/src/main/ruby/shell/commands/describe.rb

```diff
@@ -44,7 +44,8 @@ def command(table)
           puts
         end
         formatter.footer
-        if table.to_s != 'hbase:meta'
+        if admin.exists?(::HBaseQuotasConstants::QUOTA_TABLE_NAME.to_s) \
```

Comment:
Shall we print `Quota is disabled` when quota table don't exist (same way describe_namespace).

Reply:
OK, let me fix it as you said.